packages/web-react/src/components/Alert/useAlertIcon.ts

```diff
@@ -1,8 +1,9 @@
+import { ElementType } from 'react';
 import { useIconName } from '../../hooks/useIconName';
-import { SpiritAlertProps } from '../../types';
+import { EmotionColor, SpiritAlertProps } from '../../types';
 
-export function useAlertIcon({ color, iconName }: SpiritAlertProps) {
-  const iconNameValue = useIconName(color, {
+export function useAlertIcon<T extends ElementType = 'div', S = void>({ color, iconName }: SpiritAlertProps<T, S>) {
+  const iconNameValue = useIconName(color as EmotionColor<string>, {
```

Comment:
```suggestion
  const iconNameValue = useIconName(color as EmotionColor<S>, {
```

Reply:
This should be a `string` because it expects an icon name, not a color.